### PR TITLE
repo 2.11.1

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by carlocab** [comment]
```
==> /usr/local/Cellar/repo/2.11.1/bin/repo help 2>&1
repo: error: Python 2 is no longer supported; Please upgrade to Python 3.5+.
```

File: Formula/repo.rb [hunk 1]
```diff
@@ -2,12 +2,14 @@ class Repo < Formula
...
-      tag:      "v2.10",
-      revision: "352c93b68084e97ec7a3c09f0bd3f2a161290211"
+      tag:      "v2.11.1",
+      revision: "b64bec6acc0e2a835f427d8424af89dfcddaf08f"
   license "Apache-2.0"
   version_scheme 1
 
   bottle :unneeded
+  
+  depends_on "python@3.9"
```
Comment (carlocab) [inline Formula/repo.rb:12]:
```suggestion

  depends_on "python@3.9"
```

File: Formula/repo.rb [hunk 2]
```diff
@@ -2,13 +2,15 @@ class Repo < Formula
...
-      tag:      "v2.10",
-      revision: "352c93b68084e97ec7a3c09f0bd3f2a161290211"
+      tag:      "v2.11.1",
+      revision: "b64bec6acc0e2a835f427d8424af89dfcddaf08f"
   license "Apache-2.0"
   version_scheme 1
 
   bottle :unneeded
 
+  depends_on "python@3.9"
```
Comment (carlocab) [inline Formula/repo.rb:12]: Is this the right solution? Why isn't `repo` able to use the built-in python3?

Comment (chenrui333) [reply Formula/repo.rb:14]: I missed the shebang update, just added.

File: Formula/repo.rb [hunk 3]
```diff
@@ -13,6 +15,8 @@ class Repo < Formula
 
   def install
     bin.install "repo"
+    rewrite_shebang detected_python_shebang, bin/"repo"
```
Comment (carlocab) [inline Formula/repo.rb:18]: 👏

File: Formula/repo.rb (hunk 2 above)
Comment (chenrui333) [reply Formula/repo.rb:14]:
```diff
diff --git a/usr/local/Cellar/repo/2.11.1/bin/repo b/../repo.org
old mode 100755
new mode 100644
index d797bf4..8f13015
--- a/usr/local/Cellar/repo/2.11.1/bin/repo
+++ b/../repo.org
@@ -1,4 +1,4 @@
- #!/usr/local/opt/python@3.9/bin/python3
+ #!/usr/bin/env python
# -*- coding:utf-8 -*-
#
# Copyright (C) 2008 The Android Open Source Project
```

**Review by carlocab — APPROVED** [review]

**Pull request merged**